# Genome-scale metabolic models for 58 bacterial species from childrenâ€™s gut microbiota

This repository contains the current genome-scale metabolic models for 58 gut bacterial species, which are selected from gut microbiota of healthy and malnourished children.

General Concepts:

Annotated genomes based on Rapid Annotation using Subsystem Technology (RAST) sever were used for reconstructing the GEMs at Kbase platform (The U.S. Department of Energy Systems Biology Knowledgebase, https://kbase.us/).

- The basic principle is:

In this work, we applied genome scale metabolic modeling, a constraint-based modeling framework, to model the gut microbial species, which were selected from healthy and malnourished children of three countries (Malawi, Bangladesh, and Sweden). These models helped to predict the metabolic capabilities of gut microbial species and to perform comparative analysis between different children groups in health and malnutrition in the term of growth of species and secretion of health-promoting small molecules such as short chain fatty acids and amino acids.


- The models contain:

|Bacterial species | Model ID | Reactions | Metabolites| Genes |
| ------------- |:-------------:|:-------------:|:-------------:|-----:|
|Bacteroides caccae |	'kbg1379.xml'| 1029|	976|	3753|
|Bacteroides fragilis	|'kbg1616.xml'|	1038	|1003	|4862|
|Bacteroides intestinalis|	'kbg1697.xml'|	1022|	981|	4796|
|Bacteroides ovatus	|'kbg28919.xml'|	1040	|993	|6680|
|Bacteroides stercoris|	'kbg1552.xml'	|1027|	983	|3481|
|Bacteroides thetaiotaomicron|	'kbg31348.xml'	|1046|	995|	4981|
|Bacteroides uniformis	|'kbg1370.xml'|	1033	|1004	|4178|
|Bacteroides vulgatus	|'kbg31349.xml'|	1075|	1014|	4574|
|Bifidobacterium adolescentis|	'kbg1371.xml'|	934	|901|	2108|
|Bifidobacterium bifidum	|'kbg25501.xml'	|915	|899	|2028|
|Bifidobacterium breve|	'kbg31087.xml'	|950|	901	|2244|
|Bifidobacterium longum	|'kbg26583.xml'|	966|	929|	2664|
|Bifidobacterium pseudocatenulatum	|'kbg26897.xml'|	939|	917	|1981|
|Bifidobacterium pseudolongum	|'kbg239964.xml'	|893|	857|	1645|
|Catenibacterium mitsuokai	|'kbg1558.xml'|	985	|944	|2341|
|Clostridium bartlettii	|'kbg1525.xml'	|1046|	978|	2805|
|Clostridium paraputrificum	|'kbg208693.xml'|	1163|	1053|	3544|
|Clostridium ramosum	|'kbg1526.xml'|	1048	|973|	3123|
|Clostridium sp SS2 1	|'kbg1373.xml'	|1088	|1018|	3048|
|Collinsella aerofaciens	|'kbg1381.xml'	|913|	868	|2173|
|Dialister succinatiphilus|	'kbg27482.xml'|	987|	949|	2214|
|Dorea formicigenerans	|'kbg1356.xml'|	1091	|1026|	3085|
|Dorea longicatena	|'kbg1357.xml'	|977	|943|	2839|
|Enterobacter aerogenes|	'kbg_MK4.xml'	|1547	|1289|	5016|
|Enterococcus faecalis	|'kbg435.xml'|	1104|	1005|	3323|
|Escherichia coli	|'kbg1870.xml'	|1526|	1255|	4655|
|Eubacterium biforme|	'kbg1911.xml'|	936	|907|	2359|
|Eubacterium eligens	|'kbg1895.xml'	|963	|943|	2690|
|Eubacterium rectale|	'kbg2826.xml'|	1057|	1001	|3480|
|Faecalibacterium cf	|'kbg3160.xml'	|1051	|981	|2840|
|Faecalibacterium prausnitzii	|'kbg3105.xml'	|1028|	964|	3291|
|Fusobacterium mortiferum	|'kbg1673.xml'	|1108|	1025	|2635|
|Haemophilus parainfluenzae|	'kbg24173.xml'	|1150|	1023	|2052||
|Lactobacillus mucosae	|'kbg24465.xml'	|951|	893	|2074|
|Lactobacillus rhamnosus	|'kbg209856.xml'|	1108|	1025|	3022|
|Lactobacillus ruminis	|'kbg84.xml'|	1004	|957|	2317|
|Lactobacillus salivarius	|'kbg3076.xml'|	958|	905|	2269|
|Megasphaera elsdenii	|'kbg241889.xml'|	1064	|1017|	2322|
|Parabacteroides distasonis|	'kbg242180.xml'|	1160	|1080 |4889| 
|Parabacteroides johnsonii|	'kbg2144.xml'|	1071|	1035	|3958|
|Prevotella copri	|'kbg2146.xml'	|925	|904	|5637|
|Roseburia intestinalis|	'kbg2141.xml'|	1086	|1017|	4125|
|Roseburia inulinivorans	|'kbg2642.xml'|	1058|	1011|	4055|
|Ruminococcus bromii	|'kbg2829.xml'|	818	|805|	2209|
|Ruminococcus gnavus|	'kbg210975.xml'|	1129	|1041	|3973|
|Ruminococcus lactaris|	'kbg206793.xml'	|997|	966|	2965|
|Ruminococcus obeum	|'kbg2823.xml'	|1080|	1014|	3614|
|Ruminococcus sp 5 1 39BFAA	|'kbg1613.xml'|	1108|	1022	|3527|
|Ruminococcus torques|	'kbg2822.xml'	|1091	|1025|	3176|
|Staphylococcus warneri|	'kbg31042.xml'	|1252|	1098	|2507|
|Streptococcus parasanguinis	|'kbg206790.xml'|	981	|896|	2168|
|Streptococcus thermophilus	|'kbg23407.xml'|	921|	857	|2259|
|Streptococcus vestibularis	|'kbg3655.xml'|	976	|900|	2052|
|Subdoligranulum variabile	|'kbg1366.xml'|	1055|	990	|6038|
|Succinivibrio dextrinosolvens	|'kbg244674.xml'|	1046	|994|	2343|
|Veillonella atypica	|'kbg3465.xml'|	977	|930|	1932|
|Veillonella dispar	|'kbg212687.xml'|	1039	|979	|2115|
|Weissella cibaria	|'kbg28337.xml'|	999	|937|	2151|


 

This repository is administered by Manish Kumar & (manishku), and Dimitra Lappa (demilappa), Division of Systems and Synthetic Biology, Department of Biology and Biological Engineering, Chalmers University of Technology

Last update: 2017-07-07


## Installation

### Required Software:

* *_PROGRAMMING LANGUAGE/Version_*  (e.g.):
  *  You need a functional Matlab installation of **Matlab_R_2015_b**  (MATLAB 7.3 and higher)
  * The [COBRA](https://github.com/opencobra/cobratoolbox) toolbox for MATLAB. An up-to-date version from COBRA GitHub repository is strongly recommended . Add the directory to your Matlab path, instructions [here](https://se.mathworks.com/help/matlab/ref/addpath.html?requestedDomain=www.mathworks.com)

### Dependencies - Recommended Software:
* libSBML MATLAB API (version [5.13.0](https://sourceforge.net/projects/sbml/files/libsbml/5.13.0/stable/MATLAB%20interface/)  is recommended).
* Gurobi Optimizer for MATLAB (version [6.5.2](http://www.gurobi.com/registration/download-reg) is recommended). 


### Installation Instructions
* Clone [ChildrenGutMicrobialGEMs](https://github.com/SysBioChalmers/ChildrenGutMicrobialGEMs) branch from [SysBioChalmers GitHub](https://github.com/SysBioChalmers)
* Add the directory to your Matlab path, instructions [here](https://se.mathworks.com/help/matlab/ref/addpath.html?requestedDomain=www.mathworks.com)


## Contributors
- [Manish Kumar](https://www.chalmers.se/en/Staff/Pages/manishk.aspx), Chalmers University of Technology, Gothenburg Sweden
- [Dimitra Lappa]( https://www.chalmers.se/en/staff/Pages/lappa.aspx), Chalmers University of Technology, Gothenburg Sweden

## License
The MIT License (MIT)

> Copyright (c) 2017 Systems and Synthetic Biology
>
> Chalmers University of Technology Gothenburg, Sweden
>
>Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:
>
>The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
>
>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.
